server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -196,14 +196,25 @@ def validate_openshift_parameters(deployment)
           deployment.errors[:openshift_storage_size] << _("OpenShift deployments must have a storage size greater than zero")
         end
 
-        validate_storage_share(deployment, deployment.openshift_storage_host, deployment.openshift_export_path, -1, -1)
+        if deployment.openshift_storage_host.empty?
```

Comment:
I kind of made that intentional since the path is meaningless without a host. Would it be better to display errors on both if they don't exist? I wasn't sure if that is used to display something to the end user about missing fields.